File-Formats/PDF/Convert-HTML-To-PDF/aspnet-mvc.md

```diff
@@ -13,54 +13,59 @@ The Syncfusion HTML to PDF converter is a .NET library used to convert HTML or w
 
 ## Steps to convert HTML to PDF document in ASP.NET MVC
 
-Create a new C# ASP.NET Web Application (.NET Framework) project.
+1. Create a new C# ASP.NET Web Application (.NET Framework) project.
 ![convert_HtmltoPdf_ASP.NET_MVC1](htmlconversion_images/aspnetmvc1.png)
 
-Install Syncfusion.HtmlToPdfConverter.AspNet.Mvc5  NuGet package as reference to your .NET applications from NuGet.org.
+2. Install [Syncfusion.HtmlToPdfConverter.AspNet.Mvc5](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.AspNet.Mvc5)  NuGet package as reference to your .NET applications from [NuGet.org](https://www.nuget.org/).
 ![convert_HtmltoPdf_ASP.NET_MVC2](htmlconversion_images/aspnetmvc2.png)
 
-Include the following namespaces in that HomeController.cs file.
+3. Include the following namespaces in that HomeController.cs file.
```

Comment:
change it as "Include the following namespaces in the HomeController.cs file"

Reply:
content changed